packages/ui-stencil/src/components/internal/orama-chat-suggestions/orama-chat-suggestions.scss

```diff
@@ -1,13 +1,13 @@
 :host {
   display: block;
+  font-family: var(--font-primary, font('primary'));
```

Comment:
shouldn't it be global?

Reply:
Yes, we don't need this, thanks for spotting.